src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs

```diff
@@ -21,8 +21,18 @@ public Task Configure(string endpointName, EndpointConfiguration config, RunSett
         var topology = Environment.GetEnvironmentVariable("AzureServiceBusTransport.Topology", EnvironmentVariableTarget.User);
         topology = topology ?? Environment.GetEnvironmentVariable("AzureServiceBusTransport.Topology");
 
-        var transportConfig = config.UseTransport<AzureServiceBusTransport>().ConnectionString(connectionString);
-        
+        var transportConfig = config.UseTransport<AzureServiceBusTransport>();
+
+        AzureServiceBusTransportConfigContext azureServiceBusTransportConfigContext;
+        if(settings.TryGet("AzureServiceBus.TransportConfigContext", out azureServiceBusTransportConfigContext))
```

Comment:
Looks like `AzureServiceBus.TransportConfigContext` is a test specific setting. I'd rather to see a name of a setting key that reflects that. Something like `AzureServiceBus.AcceptanceTest.TransportConfigContext`?